src/rest-server/src/models/job.js

```diff
@@ -425,6 +427,8 @@ class Job {
           'jobData': data,
           'inspectFormat': '{{.State.Pid}}',
           'jobEnvs': jobEnvs,
+          'azRDMA': azureEnv.azRDMA == 'false' ? false : true,
```

Comment:
`===` might be better?

Reply:
Waht's the difference in the string compare between ```===``` and ```==```

Reply:
It works in this case since we have validation before, but never use `==` is a better spec

Reply:
Yes. So I'm really curious about in which input without validation will cause an issue. Do you have an example

Reply:
@Gerhut What's your opinion?

Reply:
BTW, seems we don't need `? :` here?

Reply:
```
> true=="false"
false
> undefined=="false"
false
> 0=="false"
false
> false=="false"
false
>
```
@mzmssg

Reply:
Replace "==" with "===" .

Reply:
test `[false] == 'false'`